package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "my-event-place",
+  "version": "1.0.0",
+  "description": "A web application for creating events",
+  "main": "index.js",
+  "repository": "https://github.com/PSIFON/my-event-place.git",
+  "author": "''",
+  "license": "MIT",
+  "scripts": {
+    "start": "npm run build && node dist/index.js",
+    "build": "rm -rf dist && babel server/ --out-dir dist/ --ignore ./node_modules",
```

Comment:
Great work @Danielshow. Please what does this line do for us in the backend?

Reply:
Since you are writing ES6 code, it is compiled to ES5 and be stored in a dist folder. That line will always remove the available `dist` folder and transpile our code to es6.